pkg/arvo/sur/group-store.hoon

```diff
@@ -1,20 +1,39 @@
+/-  *group
 |%
-+$  group  (set ship)
+::  $action: request to change group-store state
 ::
-+$  group-action
-  $%  [%add members=group pax=path]                :: add member to group
-      [%remove members=group pax=path]             :: remove member from group
-      [%bundle pax=path]                           :: create group at path
-      [%unbundle pax=path]                         :: delete group at path
-  ==
+::    %add-group: add a group
+::    %add-members: add members to a group
+::    %remove-members: remove members from a group
+::    %add-tag: add a tag to a set of ships, creating the tag if it doesn't exist
+::    %remove-tag:
+::      remove a tag from a set of ships. If the set is empty remove the tag
+::      from the group.
+::    %change-policy: change a group's policy
+::    %remove-group: remove a group from the store
+::    %groupify: unset .hidden flag
 ::
-+$  group-update
-  $%  [%initial =groups]
-      [%keys keys=(set path)]                      :: keys have changed
-      [%path members=group pax=path]
-      group-action
++$  action
+  $%  [%add-group =group-id =policy hidden=?]
+      [%add-members =group-id ships=(set ship) tags=(set tag)]
+      [%remove-members =group-id ships=(set ship)]
+      [%add-tag =group-id =tag ships=(set ship)]
```

Comment:
Not sure how to feel about tags being included in the `%add-members` action. Especially with the "members can add themselves sometimes" from a previous comment, this seems to add a lot of complexity. Think it should strictly be in `%add-tag` regardless.